src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/passbook
+spring.datasource.username=postgres
+spring.datasource.password=qwerty1201
+#spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
Read about dialect

Reply:
**- Dialect is a class and a bridge between Java JDBC types and SQL types, which contains mapping between java language data type and database datatype. 
- Dialect allows Hibernate to generate SQL optimized for a particular relational database.**